sites/docs/docs/deployment/github-pages.md

```diff
@@ -0,0 +1,24 @@
+---
+    hide_table_of_contents: false
+    title: Github Pages
+---
+
+:::caution
+Deploying to Github pages requires more javascript knowledge than alternatives like [Evidence Cloud](/deployment/evidence-cloud) [Netlify](/deployment/netlify) or [Vercel](/deployment/vercel), if you are unsure, consider using another method.
+:::
+
+## Deploying to Github Pages
+
```

Comment:
Is it worth a short para on why?

GitHub pages are typically deployed with a base path that is not `/` eg ..., which is a non standard configuration, and requires some working around.